rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    person.add_rental(self)
+    book.add_rental(self)
```

Comment:
- [ ] I can see you're adding a rental to the person but there is no add_rental method in the person class responsible for this action. Kindly create the add_rental method in the Person class so you will be able to use it here. 👍🏽
- [ ] Also, you could first add the rentals in the initialize method of the person class. `@rentals = []` and then add the add_rental method as that will help achieve the 'has-many' association. It looks like this:

```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

**Note**
>The changes suggested should be done inside the person.rb file so the method usage in this fill will work fine 👍🏽